pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/phimanifest/PhiManifestTest.java

```diff
@@ -176,13 +178,26 @@ public void phiReportTest() throws Exception {
         PhiManifest phiManifest = phiManifestService.generateDataForReport(participant, orders, ddpInstanceDto);
         oncHistoryDetail = OncHistoryDetail.getOncHistoryDetail(createdTissue.getOncHistoryDetailId(), instanceName);
         assertPhiManifest(phiManifest, orders, createdTissue, oncHistoryDetail, participant);
+
+        // now change the answers to various questions and verify that the phi manifest columns are correct
```

Comment:
The test has been augmented to verify that when key answers are blank, the PHI manifest object shows them as blank instead of `No`.